examples/audio/audio_looping.rs

```diff
@@ -0,0 +1,46 @@
+use bevy::prelude::*;
+
+/// This example illustrates how to load and play an audio file on repeat.
+
+struct OverworldThemeTimer(Timer);
+
+struct OverworldTheme {
+    track: Handle<AudioSource>,
+}
+
+const THEME_LENGTH: f32 = 13.;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(OverworldThemeTimer(Timer::from_seconds(THEME_LENGTH, true)))
+        .init_resource::<OverworldTheme>()
+        .add_startup_system(play_theme.system())
+        .add_system(repeat_theme.system())
+        .run();
+}
+
+impl FromWorld for OverworldTheme {
+    fn from_world(world: &mut World) -> Self {
+        let world = world.cell();
```

Comment:
I'm not sure we need this line? It looks like we're only accessing one part of the world at once, so we should be fine without a cell.

```suggestion
```